### DISCUSSION TITLE
After migrating maps and logging out the game, I literally cannot reopen it

### DISCUSSION THREAD

**Discussion by Tiancheng964** [ppy/osu#16318] [Q&A] [answered]
I just recently installed osu! lazer and upon opening it the first time, it was all fine. It asked me if I wanted to migrate anything and I clicked yes, then I quit the game to play something else. Upon coming back I clicked the shortcut and nothing happened, I have uninstalled and reinstalled the game countless times now but every time I open it I see it in task manager for about three seconds before it disappears. I am running Windows 10 Home 64 bit with 21H2 Version (I literally updated yesterday), Intel Core i7 6850k @ 3.60GHz, gtx1650 super, and 16GB of DDR4 ram. Can someone please help me.

**Comment by Theighlin** [comment, marked as answer]
Before anyone can help you you're gonna need to follow #15188 and post logs if there are any.

**Reply by Tiancheng964** [reply]
[database.log](https://github.com/ppy/osu/files/7806173/database.log)
[network.log](https://github.com/ppy/osu/files/7806174/network.log)
[performance.log](https://github.com/ppy/osu/files/7806176/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7806177/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7806178/updater.log)

**Reply by peppy** [reply]
Your database is unfortunately corrupt. This can happen if your PC lost power or you quit the app in a weird state.

Delete `client.db` and re-run the import. Note that it will be faster on re-run as the files have already been copied - it just needs to repopulate the database.

**Reply by Tiancheng964** [reply]
Osu! lazer still doesn't open though

**Reply by peppy** [reply]
please provide new logs then

**Reply by Tiancheng964** [reply]
How do I find logs for Osu! Lazer as the method Thighlin showed me was for Osu! standard?

**Reply by integerrr** [reply]
This whole repository is for lazer only, is this issue about lazer not opening or stable not opening? 

The logs your provided were lazer logs

**Reply by Tiancheng964** [reply]
[database.log](https://github.com/ppy/osu/files/7807220/database.log)
[network.log](https://github.com/ppy/osu/files/7807221/network.log)
[performance.log](https://github.com/ppy/osu/files/7807222/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7807223/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7807224/updater.log)

**Reply by Tiancheng964** [reply]
here are the new logs

**Reply by peppy** [reply]
It looks like the same issue. Are you sure you're deleting the correct client.db file?

**Reply by Tiancheng964** [reply]
yes I have

**Reply by Tiancheng964** [reply]
unless the db file also has other numbers after it then i've been deleting the correct one

**Reply by Tiancheng964** [reply]
![Screenshot 2022-01-04 231051](https://user-images.githubusercontent.com/97086709/148060581-fa439655-3920-42e9-b717-59fe7ad88006.png)
It's the first one right?

**Reply by integerrr** [reply]
It's what I thought - I think you're deleting the one for stable. It's not even called `client.db`

![image](https://user-images.githubusercontent.com/7279624/148062523-f0db3fd1-2d70-4168-a0fc-8bf7d3205452.jpeg)

**Comment by peppy** [comment]
client dot db.
not osu exclamation mark dot db.

**Reply by Tiancheng964** [reply]
![Screenshot 2022-01-04 232644](https://user-images.githubusercontent.com/97086709/148062412-eb8b5b42-4f81-46d0-9d1d-56535db569a6.png)

I don't see it.

**Reply by integerrr** [reply]
It's in the parent folder of where you got your logs file

**Reply by Tiancheng964** [reply]
Oh sorry my bad
